commitlint.config.js

```diff
@@ -0,0 +1 @@
+module.exports = {extends: ['@commitlint/config-angular']};
```

Comment:
Can you change this line to a `commitlint` property on `package.json`? Less one file to worry about. :)